servers/curated-corpus-api/src/database/types.ts

```diff
@@ -154,6 +154,20 @@ export type CreateCustomSectionInput = {
   disabled: boolean;
 };
 
+export type UpdateCustomSectionInput = {
+  externalId: string;
+  title: string;
+  description: string;
+  heroTitle?: string;
+  heroDescription?: string;
+  startDate: string;
+  endDate?: string;
+  scheduledSurfaceGuid?: string;
```

Comment:
`scheduledSurfaceGuid` needs to be removed as well. lets make sure that the mutation inputs match the graphql input: `UpdateCustomSectionInput`